packages/zoe/src/zoe.js

```diff
@@ -251,7 +259,7 @@ import { makeTables } from './state';
  * @param  {AmountKeywordRecord[]} newAllocations An
  * array of amountKeywordRecords  - objects with keyword keys
  * and amount values, with one keywordRecord per offerHandle.
- * @param  {Keyword[]} sparseKeywords An array of string
+ * @param  {Keyword[]=} sparseKeywords An array of string
```

Comment:
TIL the Closure syntax for optional arguments.

I prefer `[sparseKeywords]`, to avoid mixing styles of documentation (we currently have both JSDoc and Typescript), but this is legit, so feel free to leave it in place.